distributed/cli/dask_scheduler.py

```diff
@@ -110,6 +122,8 @@ def del_pid_file():
         scheduler.stop()
         if bokeh_proc:
             bokeh_proc.close()
+        if os.path.exists(local_directory):
```

Comment:
If I were misunderstanding the meaning of `--local-directory` and passing an existing directory, I would probably not like Dask to delete it. Is this necessary?